src/package-change-checker.js

```diff
@@ -3,11 +3,15 @@ const os = require('os');
 const git = require('./git');
 const loader = require('./loader');
 
-const commit1 = 'ORIG_HEAD';
-const commit2 = 'HEAD';
+let commit1 = 'ORIG_HEAD';
+let commit2 = 'HEAD';
 
-const hasChangedDependencies = () =>
-  git
+const hasChangedDependencies = (hashes = []) => {
+  if (hashes.length === 2) {
+    // assume post-checkout scenario with two hashes given as params from Git
+    [commit1, commit2] = hashes;
```

Comment:
Mutating the two module scoped vars introduces a side effect into this function (i.e. the commit defaults become lost after calling with hashes). Could you keep the two defaults as consts and create new vars local to the `hasChangedDependencies` function?